src/EnergyPlus/PlantCondLoopOperation.cc

```diff
@@ -3673,7 +3673,7 @@ void FindCompSPLoad(EnergyPlusData &state,
 
     // load local variables from the data structures
     CompMinLoad = this_component.MinLoad;
-    CompMaxLoad = this_component.MaxLoad;
+    CompMaxLoad = this_component.getDynamicMaxCapacity(state);
```

Comment:
Down at line 3763 if code that will cap MyLoad if the component cannot meet the load. Calling this function provides a more accurate full load capacity for use in plant manager calculations.

        // Check bounds on MyLoad
        if (std::abs(this_component.MyLoad) > CompMaxLoad) {
            this_component.MyLoad = sign(CompMaxLoad, this_component.MyLoad);
        }